tests/connectors/argilla/test_argilla_wrapper_client.py

```diff
@@ -390,15 +420,16 @@ def test_client_can_load_existing_dataset(
             ),
             {"multilabel": ["a", "b"]},
         ),
-        (
-            rg.RankingQuestion(name="ranking", values=["reply 1", "reply 2"]),
-            {
-                "ranking": [
-                    {"rank": 1, "value": "reply 2"},
-                    {"rank": 2, "value": "reply 1"},
-                ]
-            },
-        ),
+        ####### RankingQuestion not supported for now
```

Comment:
Do we want to write this somewhere else?

Reply:
Deleted the comment as discussed